data/private

```diff
@@ -126,22 +126,26 @@ full:instant.arubanetworks.com
 full:setmeup.arubanetworks.com
 
 # ASUS router
+full:asusrouter.com
 full:router.asus.com
+full:www.asusrouter.com
 
 # KDE
 networkcheck.kde.org
 
+# Netgear router
+full:routerlogin.com
+full:www.routerlogin.com
```

Comment:
> If your router isn't NETGEAR, then just visit this domain, it will redirect to the setup page that shows how to setup your router correctly since this domain didn't filtered by the router.
> 
> If your domain is NETGEAR, visit this domain will show the management page.

You can just show me the evidence, you know.
Since this is reviewing and I do not have the same knowledge on the background of this PR as you have. It is up to you to knowledge me till I can see the benefits of doing so, or I will not pull the trigger.
No source, no go.

Reply:
I got you, I will append references next time.

Reply:
https://kb.netgear.com/27199/I-can-t-access-my-router-what-do-I-do

This page gives you proof, and my last router is NETGEAR, the url on the label is `routerlogin.com`.

This page also shows another domain `routerlogin.net`, but not used by my model. My old NETGEAR router has been sold, and now I can’t verify whether this is also available, so I didn’t add it.

Reply:
I think it is OK to not include the domain `routerlogin.net` for now. I insist only domains that could be verified be included in this category.